pkg/cfn/manager/api.go

```diff
@@ -83,13 +91,41 @@ func (c *StackCollection) doCreateStackRequest(i *Stack, templateBody []byte, pa
 	logger.Debug("input = %#v", input)
 	s, err := c.provider.CloudFormation().CreateStack(input)
 	if err != nil {
+		logGuidanceOnAssumeRoleFailure(err, input.RoleARN)
 		return errors.Wrapf(err, "creating CloudFormation stack %q", *i.StackName)
 	}
 	logger.Debug("stack = %#v", s)
 	i.StackId = s.StackId
 	return nil
 }
 
+func logGuidanceOnAssumeRoleFailure(err error, roleArn *string) {
```

Comment:
This is an advanced use case, I would expect the user to understand very well what they are doing. I don't see much need for this helper, would you mind to omit it for now before we hear from users?

Reply:
Completely understood. Removed in 7e45c81